### DISCUSSION TITLE
How do you use the localtwpdf feature ?

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#48] [Q&A] [answered]
I tried the localtwpdf feature in tiddlyclip. How do you use this feature in your TW usage? Do you clip text from such local pdf files? Is it possible to annotate local pdf with anchor links ? By anchor links I meant links which when clicked takes us to the clipped text from pdf.

**Comment by buggyj** [comment, marked as answer]
localtwpdf is used to embed a pdf into tiddler. This is achieved listing the directory in a browser window and right clicking on the file name and acting the localpdf rule from the tiddlyclip context menu, eg for 'Current Mathematical Methods Used in QSAR QSPR Studies.pdf' I navigate to the containing folder and then right its name....

![image](https://user-images.githubusercontent.com/2855795/149401860-bdda8c13-0eeb-4c87-b73e-f2cebf34125f.png)

the result is 

![image](https://user-images.githubusercontent.com/2855795/149402516-34baa275-8332-49ca-9316-1e5f732e449c.png)

**Reply by arunnbabu81** [reply]
I was able to embed pdf as described above. My doubts were regarding these things
`Do you clip text from such local pdf files? Is it possible to annotate local pdf with anchor links ? By anchor links I meant links which when clicked takes us to the clipped text from pdf`

**Reply by buggyj** [reply]
it is not possible to annotate pdfs within the browser, there are other tools to annotate pdfs. All that tiddlyclip can do is clip text from a pdf displayed within a browser tab.
